antu/rke/cluster.yml

```diff
@@ -0,0 +1,50 @@
+---
+nodes:
+- address: antu01.ls.lsst.org
+  hostname_override: antu01
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+- address: antu02.ls.lsst.org
+  hostname_override: antu02
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+- address: antu03.ls.lsst.org
+  hostname_override: antu03
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+- address: antu04.ls.lsst.org
+  hostname_override: antu04
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+services:
+  kubelet:
+    extra_args:
+      node-status-max-images: "-1"
+      max-pods: 250
+network:
+  plugin: canal
+ssh_key_path: ~/.ssh/id_rsa
+ignore_docker_version: true
+kubernetes_version: v1.27.12-rancher1-1
```

Comment:
can this cluster start at 1.28.9 please 🙏🏻

Reply:
Done.   Please don't forget to remove the rke version string from the antu cluster when updating `ls`.